src/main/scala/ai/privado/tagger/sink/SinkArgumentUtility.scala

```diff
@@ -0,0 +1,291 @@
+package ai.privado.tagger.sink
+
+import io.shiftleft.codepropertygraph.generated.nodes.{
+  AstNode,
+  CfgNode,
+  NewFile,
+  NewTag,
+  Block,
+  Literal,
+  Call,
+  Identifier,
+  FieldIdentifier,
+  Local
+}
+import io.shiftleft.codepropertygraph.generated.{EdgeTypes, Operators}
+import io.shiftleft.semanticcpg.language._
+import io.shiftleft.utils.IOUtils
+import io.joern.dataflowengineoss.DefaultSemantics
+import io.joern.dataflowengineoss.queryengine.{EngineConfig, EngineContext}
+import io.joern.dataflowengineoss.semanticsloader.Semantics
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.JavaProperty
+
+import scala.collection.mutable.ListBuffer
+import scala.util.matching.Regex
+import scala.collection.mutable.ListBuffer
+import java.io.{PrintWriter, ByteArrayInputStream, ByteArrayOutputStream, ObjectInputStream, ObjectOutputStream}
+import java.nio.file.Files
+
+object SinkArgumentUtility {
```

Comment:
A lot of things are done in this utility functions, please add comments explaining the use of each function and descriptive comment explaining the code functionality where ever required

Reply:
Added